scripts/generate_folders_for_year.swift

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env xcrun swift
+
+//
+//  generate_folders_for_year.swift
+//  Presentation Archive
+//
+//  Created by Blake Merryman on 2018-01-12.
+//  Copyright © 2018 Nashville CocoaHeads. All rights reserved.
+//
+// 	This script generates the folder structure to house presenation materials
+//	for a specified year.
+//
+
+import Foundation
+
+var rawArguments = CommandLine.arguments
+rawArguments.removeFirst() // don't need this system provided info
+
+guard rawArguments.count == 2 else {
+  print("Generation failed. Please pass in valid arguments (e.g. --year 2017).")
+  exit(EXIT_FAILURE)
+}
+
+guard let year = Int(rawArguments[1]) else {
+  print("Generation failed. Please pass in valid year (e.g. 2017).")
+  exit(EXIT_FAILURE)
+}
+
+let months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October"]
```

Comment:
Just a thought. It might be nice if the directories are displayed in order. As it is now, it looks like GitHub sorts them alphabetically.

If you feel that is important we could prefix the month with the month number. `"01 January", "02 February", ...`

<img width="234" alt="screenshot 2018-01-13 18 16 57" src="https://user-images.githubusercontent.com/3932168/34911873-917746b8-f898-11e7-8677-fb7d3fc6192a.png">

Reply:
Since we use the month names in the README maybe we could have a separate array to hold the month numbers and then add that on when building the directory path.

Reply:
No need for a second array. Can just use `.enumerated()`.